vendor-extra/ViewsBundle/src/Views/ConvertsChildren.php

```diff
@@ -16,6 +17,10 @@ trait ConvertsChildren
 
     final protected function convertChildren(Element $object, array $context = []) : array
     {
+        if ($context['strip_inline'] ?? false) {
```

Comment:
Not sure whether we should do this or not.

Reply:
This is the implementation of https://github.com/libero/jats-support/pull/13#discussion_r294214068.